helper/common/common.go

```diff
@@ -358,6 +358,16 @@ func BigIntDivCeil(a, b *big.Int) *big.Int {
 		Div(result, b)
 }
 
+// SafeAddUint64 sums two unsigned int64 numbers if there are no overflow.
```

Comment:
"SafeAddUint64 sums two unsigned int64 numbers if there is no overflow"

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/2026/commits/76c28464ccc8971a6a08240d2c4fa3b78686d750